src/docs/exampleComponents/AreaChart/AreaChartConnectNulls.js

```diff
@@ -1,95 +1,93 @@
-import React, { PureComponent } from 'react';
+import React from 'react';
 import { AreaChart, Area, XAxis, YAxis, CartesianGrid, Tooltip, ResponsiveContainer } from 'recharts';
 
-export default class Example extends PureComponent {
+const data = [
```

Comment:
Now that you have moved the data array outside of the component, does the stackblitz button still open a working example?